### ISSUE TITLE
Order of play buttons inconsistent in the browser GUI

### ISSUE THREAD

**Issue by benoitb85** [ampache/ampache#2000]
# Issue Template

## Environment

* Ampache version: Ampache 4.0.0-develop
* Web server + version: apache2 from Debian Buster
* Server operating system: Debian Buster
* Client operating system: Windows 10 + Firefox Latest

* Client type:
  * Web

## Description

This is an improvement suggestion for the browser GUI.

The order of play buttons is:

* "Play, Play last, Play next" on the album view
* "Play, Play next, Play last" on the song view

I would suggest to make the order consistent. For example "Play, Play next, Play last" everywhere.

**Comment by kuzi-moto** [comment]
This seems like a pretty reasonable request. Sounds easy enough, I may be able to submit a pull request for this.

**Comment by lachlan-00** [comment]
Thank you @benoitb85 and @kuzi-moto i never noticed this!

But like all bugs once you see it you can't unsee. i'll try and test/merge tonight (not that it's a large change but that we should test each change.)

**Comment by lachlan-00** [comment]
sorry it took so long guys, but i just want to make sure i've tested things live first even though they look simple.

**Comment by benoitb85** [comment]
I just updated my Ampache (got a notification that a new version was available) but I don't see the change. How can I make sure that I have the fixed version installed ?

**Comment by kuzi-moto** [comment]
I think it was broken in more than one place. I fixed it on the individual song page.

![image](https://user-images.githubusercontent.com/17068090/62549525-40ff7280-b82e-11e9-9afd-fc789ecf5d03.png)

I noticed it's also an issue on the home page for "Recently Played", is that where you're talking about?

![image](https://user-images.githubusercontent.com/17068090/62549604-6e4c2080-b82e-11e9-963b-6e2591b05ef0.png)
